src/test/parse-fail/mod_file_not_exist.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// ignore-windows
```

Comment:
Thank you :-)

Ah, of course, you have to r+ it! I was wondering why it wasn't being tested again...